Content.Client/Players/PlayTimeTracking/JobRequirementsManager.cs

```diff
@@ -21,7 +21,7 @@ public sealed class JobRequirementsManager
     [Dependency] private readonly IPlayerManager _playerManager = default!;
     [Dependency] private readonly IPrototypeManager _prototypes = default!;
 
-    private readonly Dictionary<string, TimeSpan> _roles = new();
+    public readonly Dictionary<string, TimeSpan> PlayTimes = new();
     private readonly List<string> _roleBans = new();
 
```

Comment:
This should be a readonly dictionary at best as other code shouldn't be modifying it direct.